utils/dataloaders.py

```diff
@@ -0,0 +1,19 @@
+from collections import defaultdict
+from typing import Type
+
+from django.db.models import Model
+from promise import Promise
+from promise.dataloader import DataLoader
+
+
+def model_loader(model: Type[Model]):
+    class ModelLoader(DataLoader):
+        def batch_load_fn(self, object_ids):
+            objects = defaultdict(model)
```

Comment:
Seems there's something I don't understand about DataLoaders...
What would happen if some object_ids are not found?
objects.get(object_id) would return a new instance of model, what's the use case for that? If graphene asks for objects that are not found, can we just raise exception?

Reply:
If the object is not found, it would return `None`, since we're filtering the objects, not explicitly `get`-ing. 

`objects.get(object_id)` would return the value that was stored on this `objects` dict, so in the end, the method calling the dataloader (e.g. the `resolve_harbor`) would be the one which would have to deal with the `None`, same way as it would with a plain `Harbor.objects.get()`.